src/javascript/app/common/form_validation.js

```diff
@@ -208,6 +208,10 @@ const Validation = (() => {
     // --------------------
     const checkField = (field) => {
         if (!field.$.is(':visible') || !field.validations) return true;
+        if (field.clearResErrorOnInput) {
```

Comment:
generally only function names are camel case, for others we should use lowercase separated by underline